inventointi_listat.php

```diff
@@ -47,17 +47,19 @@
 
 // selitetark   = n�ytett�v�t monivalintalaatikot, jos tyhj��, otetaan oletus alhaalla
 // selitetark_2 = mitk� n�ytett�vist� monivalintalaatikoista on normaaleja alasvetovalikoita
-$query = "SELECT selitetark, REPLACE(selitetark_2, ', ', ',') selitetark_2
+$query = "SELECT selite, selitetark, REPLACE(selitetark_2, ', ', ',') selitetark_2
           FROM avainsana
           WHERE yhtio  = '$kukarow[yhtio]'
-          AND laji     = 'HAE_JA_SELAA'
+          AND laji     = 'INVLISTA_OSTRY'
```

Comment:
@samiluukkonen @tonyselkamo Onko tämä taaksepäin yhteensopiva? Aika monella on käytössä "HAE_JA_SELAA"-avainsanoja.